lib/ruby_lsp/listeners/code_lens.rb

```diff
@@ -151,23 +149,21 @@ def on_call_node_enter(node)
           return
         end
 
-        if [DESCRIBE_KEYWORD, IT_KEYWORD].include?(name)
-          case name
-          when DESCRIBE_KEYWORD
-            add_spec_code_lens(node, kind: :group)
-            @group_id_stack.push(@group_id)
-            @group_id += 1
-          when IT_KEYWORD
-            add_spec_code_lens(node, kind: :example)
-          end
+        case name
+        when :describe
+          add_spec_code_lens(node, kind: :group)
+          @group_id_stack.push(@group_id)
+          @group_id += 1
+        when :it, :specify # `specify` is an alias for `it`
```

Comment:
Do we need the comment? It doesn't add much that the code doesn't tell us already.

Reply:
I think it's helpful. The code shows that we treat them the same, but the the comment explains _why_.